python/cudf/setup.py

```diff
@@ -43,6 +44,52 @@
 except Exception:
     nthreads = 0
 
+cmdclass = versioneer.get_cmdclass()
+
+
+class build_ext_and_proto(build_ext):
+    def run(self):
+        # Get protoc
+        protoc = None
+        if "PROTOC" in os.environ and os.path.exists(os.environ["PROTOC"]):
+            protoc = os.environ["PROTOC"]
+        else:
+            protoc = find_executable("protoc")
+        if protoc is None:
+            sys.stderr.write("protoc not found")
+            sys.exit(1)
+
+        # Build .proto file
+        for source in ["cudf/utils/metadata/orc_column_statistics.proto"]:
+            output = source.replace(".proto", "_pb2.py")
+
+            if not os.path.exists(output) or (
+                os.path.getmtime(source) > os.path.getmtime(output)
+            ):
+                sys.stderr.write("compiling " + source + "\n")
```

Comment:
I don't think we should be writing to `stderr` for logging. Maybe just remove this line?

Reply:
Done, also fixed gitignore to properly ignore generated `_pb2.py` files